src/index.js

```diff
@@ -200,6 +207,21 @@ const handleFooterElementOnClick = (tabName) => {
         document.getElementById(tabId).disabled = tabId === alfa
     })
 }
+
+const removeElementsOnTabChange = (tabName) => {
+    const allTabIds = [
```

Comment:
And there here instead on manualy creating array, you should filter for such ids automatically.

You need to in order:
1. Update all keys for tab-wrapper-related element to contain distinct phase.
2. Convert this object to array
3. Filter array to tabs only
4. Done
5. Then you could create your beta and do what you do there.
6. After beta is done, you need to use beta.forEach to remove all children from each element